packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -81,12 +93,13 @@ export function getProgressColor(
   }
 
   return palette.status.ok;
-}
+};
 
 /** @public */
 export function Gauge(props: Props) {
```

Comment:
Would prefer to properly export this so that we can see the change in the API report, and also tweak the naming a bit alongside it. Could you rename the `Props` to `GaugeProps` and export it from the package?